packages/ui/ClientLogo/index.tsx

```diff
@@ -43,6 +43,11 @@ export const ClientLogo = ({ environmentId, product, previewSurvey = false }: Cl
       ) : (
         <Link
           href={`/environments/${environmentId}/settings/lookandfeel`}
+          onClick={(e) => {
+            if (!environmentId) {
```

Comment:
Here link is actually the Add logo button

<img width="169" alt="Screenshot 2024-04-29 at 12 41 52 PM" src="https://github.com/formbricks/formbricks/assets/67850763/ad9d79b6-8524-4b35-bd48-e302bc8eea23">

And when environmentId is not present, which is the case on look and feel page preview, there it would be better that we still show the `Add Logo` Link which on click scrolls down to `Add Logo` settings. What do you think ?🤔


https://github.com/formbricks/formbricks/assets/67850763/af7f3465-7ac8-4c51-8460-d11a4f4af9fb